CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2017-06-07 - Release 2.0.0
+
+- Data for CentOS and epel yum repositories added.
+- Yum repositories can now be configured via managed_repos and repos parameters. 
+  By default these are not configured.
+- More precise validation of versionlock strings.
+- New config_options hash parameter allows yum configuration to be specified. See
+  README.md for examples.
+- Main class parameters keepcache, debuglevel, exactarch, obsoletes, gpgcheck and
+  installonly_limit are removed. Use config_options hash instead.
```

Comment:
@traylenator can you try the new 'bundle exec rake changelog' to generate the changelog?